media/css/plugincheck/plugincheck.less

```diff
@@ -246,6 +245,48 @@ html[dir="rtl"] {
 .footer-newsletter-form {
     padding: 40px 20px;
 }
+.warning {
+    .span(8);
```

Comment:
As this is a `<p>` element, you can probably just avoid setting a `width` on it entirely as it's block level and will fit the content of its parent. This will mean you don't need to set any additional media queries too :+1: